era5cli/cli.py

```diff
@@ -328,7 +339,9 @@ def _execute(args):
                             statistics=statistics,
                             pressurelevels=args.levels,
                             threads=args.threads,
-                            merge=args.merge)
+                            merge=args.merge,
+                            preliminary_back_extension=
+                            args.preliminary_back_extension)
```

Comment:
```suggestion
                            args.preliminary_back_extension)
```
You can dedent the arguments, so they are only 1 level indented w.r.t. the function call